components/Dialogs/src/MDCAlertControllerView.h

```diff
@@ -26,6 +26,8 @@
 @property(nonatomic, strong, nullable) UIColor *buttonColor UI_APPEARANCE_SELECTOR;
 @property(nonatomic, strong, nullable) UIColor *buttonInkColor UI_APPEARANCE_SELECTOR;
 
+@property(nonatomic, assign) CGFloat cornerRadius UI_APPEARANCE_SELECTOR;
```

Comment:
Do we want to extend UIAppearance support? Since this property is settable on the controller, I don't think it's absolutely necessary for styling.  

cc/ @jverkoey since @ianegordon  is OOO this week.

Reply:
Just note that corner radius is also extended on MDCCard. Are dialogs different?

Reply:
When that Cards API was written, I believe we still supported UIAppearance. The commit [history shows that `cornerRadius` was added on February 1, 2018](https://github.com/material-components/material-components-ios/blame/develop/components/Cards/src/MDCCard.h#L27).   UIAppearance [support was dropped](https://github.com/material-components/material-components-ios/pull/3212) [somewhere in mid-March](https://www.pivotaltracker.com/n/projects/2150420/stories/155533126/comments/187416781).

Reply:
Okay. Should we remove the UI_APPEARANCE_SELECTOR from all other MDCAlertControllerView properties, like titleFont, titleColor, etc. ?